test/FsAutoComplete.Tests.Lsp/DependentFileCheckingTests.fs

```diff
@@ -27,12 +27,12 @@ let tests state =
         Expect.isEmpty bDiags $"There should be no diagnostics in {bFile}"
         let bDiagsStream = bDoc.CompilerDiagnostics
         // make a change to A (that is clearly incorrect)
-        let! startAChange = Document.changeTextTo "farts" aDoc |> Async.StartChild
+        let! startAChange = Document.saveText "farts" aDoc |> Async.StartChild
```

Comment:
Previously the behavior here was to check all dependent files on every change. However in practice this can be a performance problem. Best trade-off I made for Adaptive's behavior is:

`DidChange` -> Check only the file that changed
`DidSave` -> Check this file and all the dependent files

This at least saves your computer from trying to type check the world on every keystroke and rather only on saves.